include/mbedtls/asn1.h

```diff
@@ -604,8 +600,10 @@ int mbedtls_asn1_get_alg_null( unsigned char **p,
  *
  * \return      NULL if not found, or a pointer to the existing entry.
  */
-const mbedtls_asn1_named_data *mbedtls_asn1_find_named_data( const mbedtls_asn1_named_data *list,
-                                       const char *oid, size_t len );
+const mbedtls_asn1_named_data *mbedtls_asn1_find_named_data(
+    const mbedtls_asn1_named_data *list,
```

Comment:
Why does this indentation differ so much compared to line 590?

Reply:
`foo(bar` continuing on the next line aligns with `bar`, to the right of the opening parenthesis. `foo(` with nothing on the same line after the opening parenthesis starts one level of indendation, unrelated to how far right the opening parenthesis is. This is a very common property of indentation styles in many languages, which we're already mostly following.